scalikejdbc-streams/src/test/resources/jdbc.properties

```diff
@@ -0,0 +1,4 @@
+driverClassName=org.h2.Driver
+url=jdbc:h2:mem:scalikejdbc;LOG=0;CACHE_SIZE=65536;LOCK_MODE=0;UNDO_LOG=0
+user=
+password=
```

Comment:
sorry, I mistook...
Should I delete `scalikejdbc-streams/src/test/resources/*.properties` and add `cp` after [this line](https://github.com/scalikejdbc/scalikejdbc/blob/636a53ea99b6764e8fb907b216b63e6db7066186/travis_before.sh#L10)?

(Actually, it is necessary to [prepare a publisher that returns Integer.MAX_VALUE records](https://github.com/reactive-streams/reactive-streams-jvm/blob/v1.0.0/tck/src/main/java/org/reactivestreams/tck/PublisherVerification.java#L1030), but it seems sufficient that at least 20,000 records, because the TCK test doesn't need them all. Therefore, we probably do not need to specify options such as UNDO_LOG = 0, so we can use `scalikejdbc-core/src/test/resources/*.properties`.)